package.json

```diff
@@ -37,7 +37,7 @@
     "file": "^0.2.2",
     "gavel": "^0.5.3",
     "glob": "^7.0.5",
-    "html": "0.0.10",
+    "apiaryio/commonjs-html-prettyprinter": "",
```

Comment:
This can be valid version or publish npm package
`"apiaryio/commonjs-html-prettyprinter": "https://github.com/apiaryio/commonjs-html-prettyprinter",`